react/Pill/Pill.less

```diff
@@ -0,0 +1,87 @@
+@import (reference) '~seek-style-guide/theme';
+
+@staticTagHeight: @row-height * 5;
+@interactiveTagHeight: @row-height * 8;
+@removeCircleDiameter: @row-height * 4;
+
+.staticPill {
+  .rawText(@interaction-type-scale);
+
+  align-items: center;
+  display: inline-flex;
+  color: @sk-charcoal;
+  background: @sk-gray-light;
+  border-radius: @staticTagHeight / 2;
+  height: @staticTagHeight;
+  line-height: @staticTagHeight;
+  padding: 0 15px;
+  margin: @row-height 10px @row-height 0;
+}
+
+.interactivePill {
+  align-items: center;
+  display: inline-flex;
+  font-size: 18px;
+  background: @sk-gray-light;
+  border-radius: 24px;
+  height: @interactiveTagHeight;
+  line-height: @interactiveTagHeight;
+  padding: 0 0 0 24px;
+  margin: @row-height 10px @row-height 0;
+}
+
+.removeButton {
+  margin: 0;
+  padding: 0;
+  height: @interactiveTagHeight - 2;
+  width: @interactiveTagHeight - 2;
+  color: @sk-white;
+  border-radius: 0;
```

Comment:
This is the default right? Im not sure its needed.

Reply:
Good catch, it's not needed. But in the process of double checking that I stumbled across this https://github.com/necolas/normalize.css/issues/706

Reply:
Interesting, we could always add that into the reset sheet if needed